dotcom-rendering/src/components/Disclaimer.tsx

```diff
@@ -8,20 +10,56 @@ const disclaimerStyles = css`
 	sup {
 		font-size: 85%;
 	}
-
 	margin-bottom: 16px;
+	color: inherit;
+	text-decoration: none;
+`;
+
+const disclaimerInlineStyles = css`
```

Comment:
This styling was ~lifted~ inspired by the styling of `RichLink.tsx`

Are there better css custom properties for a generic inline component?

Reply:
I don't think so but we could probably rename them to uncouple the var name a bit?